shell/common/common_benchmarks.sh

```diff
@@ -172,22 +172,22 @@ benchmark_suite_start_config() {
   source_file "$ALOJA_REPO_PATH/shell/common/common_java.sh"
   set_java_requires
 
-  if [ ! -z "$use_hadoop" ]; then
+  if [ "$use_hadoop" ]; then
```

Comment:
That doesn't do anything to fix the case where `$use_hadoop` is set but has a value like `0` or `false`. It's just a more succinct way of writing what you already had. I was thinking that maybe a simple `1` or `0` would be sufficient, then we explicitly test against those values: `[[ $use_hadoop = 1 ]]`